bentoml/_internal/bento/build_config.py

```diff
@@ -246,9 +320,31 @@ def with_defaults(self) -> "DockerOptions":
     def write_to_bento(
         self, bento_fs: "FS", build_ctx: str, conda_options: "CondaOptions"
     ) -> None:
+        """
+        Convert all user-provided options to respective file under a 🍱 directory.
+
+        Args:
+            bento_fs (:code:`fs.base.FS`):
+                The filesystem representing a bento directory.
+            build_ctx (:code:`str`):
+                The path to the given build context. This is the context directory of the :code:`bentofile.yaml`.
```

Comment:
https://docs.bentoml.org/en/latest/concepts/bento.html#advanced-project-structure

Build context is your Python project’s working directory. This is from where you start the Python interpreter during development so that your local python modules can be imported properly. Default to current directory where the bentoml build takes place.